controllers/review_controller.py

```diff
@@ -0,0 +1,96 @@
+from flask import Blueprint, jsonify
+from models.model import Review, db
+from flask import request
+
+
+
+# all urls  (user paths):
+   # Get method: /api/v1/review/<int:user_id>/<int:mobile_id>   (to get the ratings given by a user for a mobile)
+   # Post method: /api/v1/review/<int:user_id>/<int:mobile_id>  (to give the ratings by a user for a mobile)
+   # Delete method: /api/v1/review/<int:user_id>/<int:review_id>   (to delete a particular review bsed on user_id and review_id)
+
+
+reviews_blueprint = Blueprint('reviews', __name__, url_prefix='/api/v1/')
```

Comment:
base url should be common and should be refrenced from a single point for all controllers